phoenix-kafka-base/pom.xml

```diff
@@ -26,8 +26,9 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.apache.phoenix</groupId>
-    <artifactId>phoenix-connectors</artifactId>
+    <artifactId>phoenix-connectors-base</artifactId>
     <version>6.0.0-SNAPSHOT</version>
+    <relativePath>../phoenix-connectors-base/pom.xml</relativePath>
```

Comment:
I'd prefer that we relocate the directories to to follow the maven structure, instead of hacking it around like this.